src/olympia/reviewers/tests/test_views.py

```diff
@@ -4816,6 +4816,13 @@ def test_abuse_reports(self):
 
         assert doc('.abuse_reports').text().split('\n') == expected
 
+        self.addon.update(status=amo.STATUS_DELETED)
+        response = self.client.get(self.url)
```

Comment:
after `self.addon.delete()` you'll need to regenerate `self.url` because the url uses `slug`, and `slug` is cleared during delete - the url will have to use `id` (or `pk`) instead of `slug`.
`self.url = reverse('reviewers.review', args=[self.addon.id])`